silk-react-components/src/ExecutionReport/ExecutionReport.jsx

```diff
@@ -1,142 +1,136 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import {Card, CardTitle, CardContent, Table, TableHead, TableBody, TableRow, TableCell} from '@eccenca/gui-elements';
-import silkStore from "../api/silkStore";
+import { Card, CardContent, CardTitle, Table, TableBody, TableCell, TableHead, TableRow } from '@eccenca/gui-elements';
 import MappingsTree from '../HierarchicalMapping/Components/MappingsTree';
-import hierarchicalMappingChannel from "../HierarchicalMapping/store";
+import { setApiDetails } from "../HierarchicalMapping/store";
 
```

Comment:
@robertisele what is the executionreport good for/ how does it is used? We removed big parts of HM testing here as it only worked with special parameters injected to the code triggering lines/functions which were never used in production mode.

Instead we wrote unit tests for the components. Are they equivalent?